### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by marclaporte** [comment]
I am a little nervous about stability of Cypht 2.0.1 vs 1.4.x but we will quickly fix any regressions:
https://github.com/YunoHost-Apps/cypht_ynh/issues/63

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
Time zone and language variables defined in the `.env` file are not set by default by the application 🤔

**Comment by marclaporte** [comment]
> Time zone and language variables defined in the `.env` file are not set by default by the application

@Shadow243: Thank you for contributing this: https://github.com/cypht-org/cypht/pull/823

Any wisdom here?

Thanks!

**Comment by ericgaspar** [comment]
!testme

**Comment by Shadow243** [comment]
> > Time zone and language variables defined in the `.env` file are not set by default by the application
> 
> @Shadow243: Thank you for contributing this: [cypht-org/cypht#823](https://github.com/cypht-org/cypht/pull/823)
> 
> Any wisdom here?
> 
> Thanks!

This seems true, we will have to correct it.

https://github.com/cypht-org/cypht/blob/master/lib/config.php#L106
https://github.com/cypht-org/cypht/blob/master/index.php#L34

File: scripts/upgrade [hunk 1]
```diff
@@ -67,22 +32,29 @@ chown -R $app:www-data "$install_dir"
...
 #=================================================
 # SPECIFIC UPGRADE
 #=================================================
 # UPDATE A CONFIG FILE
 #=================================================
-#ynh_script_progression --message="Updating a configuration file..." --weight=1
+ynh_script_progression --message="Adding a configuration file..." --weight=1
 
-#ynh_add_config --template="../conf/hm3.sample.ini" --destination="$install_dir/hm3.ini"
+ynh_add_config --template=".env.example" --destination="$install_dir/.env"
```
Comment (alexAubin) [inline scripts/upgrade]:
```suggestion
ynh_add_config --template=".env" --destination="$install_dir/.env"
```

File: scripts/install [hunk 2]
```diff
@@ -66,10 +55,10 @@ ynh_install_composer --install_args="--ignore-platform-reqs"
 #=================================================
 ynh_script_progression --message="Adding a configuration file..." --weight=1
 
-ynh_add_config --template="../conf/hm3.sample.ini" --destination="$install_dir/hm3.ini"
+ynh_add_config --template=".env.example" --destination="$install_dir/.env"
```
Comment (alexAubin) [inline scripts/install]:
```suggestion
ynh_add_config --template=".env" --destination="$install_dir/.env"
```

**Comment by marclaporte** [comment]
> This seems true, we will have to correct it

@Shadow243 Any updates?

**Comment by ericgaspar** [comment]
!testme

File: conf/.env [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+DB_SOCKET=/var/lib/mysqld/mysqld.sock
+
+SESSION_TYPE=PHP
+AUTH_TYPE=DB
+
+IMAP_AUTH_NAME=localhost
+IMAP_AUTH_SERVER=localhost
+IMAP_AUTH_PORT=143
+IMAP_AUTH_TLS=
+IMAP_AUTH_SIEVE_CONF_HOST=
```
Comment (Salamandar) [inline conf/.env:19]: Maybe set localhost here

**Review by Salamandar — CHANGES_REQUESTED** [review]

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**